backend/app/Models/Specialization.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+class Specialization extends Model
```

Comment:
Please provide additional information about model's properties in docblocks. You can find an example here: https://github.com/blumilksoftware/toby/blob/main/app/Eloquent/Models/Holiday.php#L13